src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -303,13 +303,48 @@ private HttpContent GetBody(string referenceId, bool schemaLessBody, Dictionary<
         }
 
         public async Task<FormulaValue> DecodeResponseAsync(HttpResponseMessage response, bool throwOnError = false)
-        {
+        {            
             var text = response?.Content == null
                             ? string.Empty
                             : await response.Content.ReadAsStringAsync().ConfigureAwait(false);
 
             var statusCode = (int)response.StatusCode;
 
+#if RECORD_RESULTS
```

Comment:
Is this intended to check in; or just temporary debugging code?

Reply:
I think it's safe to check in as this code will never end up in a nuget. 
It's only there for intercepting network results after a connector call & facilitate debugging/the creation of tests.